devicetypes/axis/axis-gear.src/axis-gear.groovy

```diff
@@ -0,0 +1,200 @@
+metadata {
+    definition (name: "AXIS Gear", namespace: "axis", author: "AXIS Labs") {  
```

Comment:
Did you intend to checkin this new DTH or only the changes to the current DTH?

Reply:
removed the old device handler. only the current DTH should have changes applied to it